src/external_cli/cargo/metadata.rs

```diff
@@ -14,12 +14,12 @@ pub(crate) fn command() -> Command {
     command
 }
 
-/// Try to obtain the Cargo metadata of this pacakge.
+/// Try to obtain the Cargo metadata of this package.
 pub(crate) fn metadata() -> anyhow::Result<Metadata> {
     metadata_with_args::<[&str; 0], &str>([])
 }
 
-/// Try to obtain the Cargo metadata of this pacakge.
+/// Try to obtain the Cargo metadata of this package.
```

Comment:
I just hit the merge button for #140, which also fixes this, so you'll get merge conflicts on these changes eventually.